site/docs/v1/tech/apis/users.adoc

```diff
@@ -762,6 +762,24 @@ The link:/docs/v1/tech/reference/data-types/#instants[instant] this login occurr
 [field]#logins``[x]``.ipAddress# [type]#[String]#::
 The IP address if provided during the login request.
 
+[field]#logins``[x]``.location.city# [type]#[String]# [since]#Available since 1.30.0#::
+The city where the login request originated.
+
+[field]#logins``[x]``.location.country# [type]#[String]# [since]#Available since 1.30.0#::
+The country where the login request originated.
+
+[field]#logins``[x]``.location.latitude# [type]#[Double]# [since]#Available since 1.30.0#::
+The latitude where the login request originated.
+
+[field]#logins``[x]``.location.longitude# [type]#[Double]# [since]#Available since 1.30.0#::
+The longitude where the login request originated.
+
+[field]#logins``[x]``.location.region# [type]#[String]# [since]#Available since 1.30.0#::
+The geographic location where the login request originated.
+
+[field]#logins``[x]``.location.zipcode# [type]#[String]# [since]#Available since 1.30.0#::
+The zipcode where the login request originated.
+
```

Comment:
These fields are only returned when you are using the Threat Detection feature. I think we have some wording in other places to designate fields or features that are only included with a particular license or edition.

Reply:
You want `docs/v1/tech/shared/_premium-edition-blurb-api.adoc`